app/presenters/tree_builder_action.rb

```diff
@@ -7,7 +7,7 @@ def tree_init_options(_tree_name)
 
   # level 0 - root
   def root_options
-    [t = N_("All Actions"), t]
+    [t = _("All Actions"), t]
```

Comment:
Well, I had to go through all those `root_options` to change the images. And *really* wanted to not have to spend another hour going through it twice, for 2 PRs.

But you're both right, it's bad form, I'll take that time and do a separate PR, possibly after Xmas though.